provider/tcp/src/main/java/org/apache/aries/rsa/provider/tcp/TCPProvider.java

```diff
@@ -32,12 +32,22 @@
 import org.apache.aries.rsa.spi.IntentUnsatisfiedException;
 import org.apache.aries.rsa.util.StringPlus;
 import org.osgi.framework.BundleContext;
+import org.osgi.service.component.annotations.Component;
 import org.osgi.service.remoteserviceadmin.EndpointDescription;
 import org.osgi.service.remoteserviceadmin.RemoteConstants;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 @SuppressWarnings("rawtypes")
+@org.osgi.annotation.bundle.Capability( //
```

Comment:
why don't you import Capability annotation?

Reply:
Makes sense to import